altair/widget/__init__.py

```diff
@@ -0,0 +1,104 @@
+import anywidget  # type: ignore
```

Comment:
oh interesting, are anywidget's types not working with mypy? maybe I forget a `py.typed`?